tdrs-backend/docker-compose.yml

```diff
@@ -99,6 +99,7 @@ services:
       ./gunicorn_start.sh && celery -A tdpservice.settings worker -l info"
     ports:
       - "5555:5555"
+    tty: true
```

Comment:
What is this for?

Reply:
If you don't enable tty in docker, then the logs don't show up with their respective colors since the default stdin/stdout streams don't support it.

Reply:
looks like the problem is stderr in term doesn't support the color and adding tty is the solution. Good job 👍